.editorconfig

```diff
@@ -0,0 +1,20 @@
+# editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+# Markdown syntax specifies that trailing whitespaces can be meaningful,
+# so let’s not trim those. e.g. 2 trailing spaces = linebreak (<br />)
+# See https://daringfireball.net/projects/markdown/syntax#p
+[*.md]
+trim_trailing_whitespace = false
+
+# Disable trailing whitespace removal in diff files,
+# where whitespace is meaningful
+[*.diff]
```

Comment:
What diff files though?

Reply:
Nothing in the actual repo, but this is is my standard config that I copy about everywhere, so it's easier to leave in :)

The diff stuff only really crops up when I'm creating temporary files when testing build output stuff